Q1. What are the three measures of central tendency?

The three measures of central tendency are:
Mean
Median
Mode

Q2. What is the difference between the mean, median, and mode? How are they used to measure the
central tendency of a dataset?

Mean: The mean is the arithmetic average of a set of numbers. It is calculated by summing all the numbers in the dataset and dividing by the number of observations. The mean is a useful measure of central tendency because it takes into account all of the data points in the dataset. However, it can be sensitive to outliers and is not always a good representation of the "typical" value in a dataset.

Median: The median is the middle value in a dataset when the data is arranged in order. The median is a useful measure of central tendency because it is less sensitive to outliers than the mean. The median is particularly useful when the data is skewed or has extreme values.

Mode: The mode is the value that appears most frequently in a dataset. The mode is a useful measure of central tendency when there is a clear "peak" or mode in the distribution of t

Q3. Measure the three measures of central tendency for the given height data:
[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [6]:
import numpy as np
import statistics as st

In [7]:
data =[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]
np.mean(data)

177.01875

In [9]:
np.median(data)

177.0

In [10]:
st.mode(data)

178

Q4. Find the standard deviation for the given data:
[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [11]:
df = [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]
np.std(df)

1.7885814036548633

Q5. How are measures of dispersion such as range, variance, and standard deviation used to describe
the spread of a dataset? Provide an example.

Measures of dispersion, such as range, variance, and standard deviation, are used to describe the degree to which a dataset's values are spread out from the central tendency (mean, median, or mode). 

The range is the simplest measure of dispersion and is calculated by subtracting the minimum value from the maximum value in a dataset. However, range can be affected by outliers, which may skew the results.

Variance measures how far the data points in a set are from the mean. It is calculated by subtracting each value from the mean, squaring the result, and then dividing the sum of the squares by the total number of data points. Variance can be expressed as a formula: 
Variance = (Σ (xi - μ)^2) / N, where xi is the data point, μ is the mean, and N is the number of data points.

Standard deviation is the square root of variance, and it is a popular measure of dispersion because it is easy to interpret and provides a useful indication of how much variation is present in a dataset. Standard deviation can be expressed as a formula: 
Standard Deviation = √(Σ(xi - μ)^2 / N), where xi is the data point, μ is the mean, and N is the number of data points.

For example, consider the following dataset of daily temperatures in Celsius: 23, 24, 25, 22, 23, 24, 26, 21, 23, 25. The mean temperature is (23+24+25+22+23+24+26+21+23+25)/10 = 23.6. 

The range of the dataset is 26-21 = 5. 

The variance can be calculated by subtracting each value from the mean, squaring the result, and then dividing the sum of the squares by the total number of data points: 
(23-23.6)^2 + (24-23.6)^2 + (25-23.6)^2 + (22-23.6)^2 + (23-23.6)^2 + (24-23.6)^2 + (26-23.6)^2 + (21-23.6)^2 + (23-23.6)^2 + (25-23.6)^2 / 10 = 3.24.

The standard deviation is the square root of the variance, which is √3.24 = 1.8. Therefore, we can say that the daily temperature in this dataset varies by approximately 1.8 degrees Celsius from the mean temperature of 23.6 degrees Celsius.

Q6. What is a Venn diagram?

A Venn diagram is a graphical representation of the relationships between sets of data. It is made up of two or more overlapping circles, each representing a set of data, and the overlapping regions represent the areas of overlap or commonality between the sets.

The diagram was first introduced by John Venn, a British mathematician, in the late 1800s, and it has since been used in a wide range of fields, including mathematics, statistics, logic, and computer science.

Q7. For the two given sets A = (2,3,4,5,6,7) & B = (0,2,6,8,10). Find:
(i) A B = (2,6)
(ii) A ⋃ B =(0,2,3,4,5,6,7,8,10)

Q8. What do you understand about skewness in data?

Skewness is a measurement of the distortion of symmetrical distribution or asymmetry in a data set. Skewness is demonstrated on a bell curve when data points are not distributed symmetrically to the left and right sides of the median on a bell curve.

Q9. If a data is right skewed then what will be the position of median with respect to mean?

If the distribution of data is skewed to the right, the mode is often less than the median, which is less than the mean.

Q10. Explain the difference between covariance and correlation. How are these measures used in
statistical analysis?

Covariance is an indicator of the extent to which 2 random variables are dependent on each other. A higher number denotes higher dependency. Correlation is a statistical measure that indicates how strongly two variables are related. The value of covariance lies in the range of -∞ and +∞.

Q11. What is the formula for calculating the sample mean? Provide an example calculation for a dataset.

The general formula for calculating the sample mean is given by 
x̄ = ( Σ xi ) / n. 
Here, x̄ represents the sample mean, xi refers all X sample values and n stands for the number of sample terms in the data set.

Example 1: Find the sample mean of 65, 59, 209, 40.

Solution:

To find: Sample mean

Sum of terms =  65 + 59 + 208 + 40 = 372 

Number of terms = 4

Using sample mean formula,

mean = (sum of terms)/(number of terms)

mean = 372/4 = 93

Q12. For a normal distribution data what is the relationship between its measure of central tendency?

Mean, Mode, Median all three are nearly equal in case of normal distrubution.

Q13. How is covariance different from correlation?

Covariance shows you how the two variables differ, whereas correlation shows you how the two variables are related.

Q14. How do outliers affect measures of central tendency and dispersion? Provide an example.

In case of mean outliers shift central tendency to a large unit but it do not affect median .

set = [1,2,3,4,5,100]

mean = 19.5

median = 3.5

so median is not affected but mean is highly affected and not following the distribution.